target/linux/ramips/dts/mt7621_zbtlink_zbt-wg1608-32m.dts

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621_zbtlink_zbt-wg1608.dtsi"
+
+/ {
+	compatible = "zbtlink,zbt-wg1608-32m", "zbtlink,zbt-wg1608-16m", "zbtlink,zbt-wg1608", "mediatek,mt7621-soc";
+	model = "Zbtlink ZBT-WG1608 (32M)";
+};
+
+&firmware {
+	reg = <0x50000 0x1fb0000>;
```

Comment:
Usually 32M Flash will cause reboot failure, need to add `broken-flash-reset;` see 74516f4357d281f093f0daac01c4c5c239acc443. It depends on the actual situation.

Reply:
This device is using W25Q256JV, but using the WSON-8 package without a hardware RESET pin. I added `broken-flash-reset;`.